src/Select/Select.scss

```diff
@@ -32,5 +31,41 @@
     &:focus {
       box-shadow: inset 0 0 0 1px $primaryLight;
     }
+
+    &.displayPlaceholder {
```

Comment:
Could it be named `dfo-select__placeholder` to express that it lives within the Select css component? This is in keeping with [the BEM framework](https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/).